sentry-jul/src/main/java/io/sentry/jul/SentryHandler.java

```diff
@@ -200,7 +202,20 @@ SentryEvent createEvent(final @NotNull LogRecord record) {
       mdcProperties =
           CollectionUtils.filterMapEntries(mdcProperties, entry -> entry.getValue() != null);
       if (!mdcProperties.isEmpty()) {
-        event.getContexts().put("MDC", mdcProperties);
+        if (!options.getMdcTags().isEmpty()) {
+          for (final String mdcTag : options.getMdcTags()) {
+            // if mdc tag is listed in SentryOptions, apply as event tag
+            if (mdcProperties.containsKey(mdcTag)) {
+              event.setTag(mdcTag, mdcProperties.get(mdcTag));
+              // remove from all tags applied to logging event
+              mdcProperties.remove(mdcTag);
```

Comment:
@bruno-garcia do we want to add entries only to either tags or context not both?